app/components/quick-filter.js

```diff
@@ -57,7 +75,9 @@ export default Component.extend({
     this._super(...arguments);
     this.set('dummyLocation', this.location);
     this.set('dummyName', this.eventName);
-    if (this.dummyName || this.dummyLocation) {this.set('disableClear', false)}
+    if (this.dummyName || this.dummyLocation) {
```

Comment:
Why ?

Reply:
This change and all the above ones are occurring automatically on saving, how can I fix this?